src/curses.rs

```diff
@@ -201,6 +208,9 @@
     /// Initialize a new `Term` instance, prepopulating the values of all the curses string
     /// capabilities we care about in the process.
     fn new(db: terminfo::Database) -> Self {
+        // todo: how to do feature detection of OSC 133?
+        let support_osc133 = false;
```

Comment:
We can probably enable it unconditionally.
I'm not aware of a relevant terminal that doesn't have basic xterm compatibility.
We can add a knob to specify xterm compatibility that or turn off OSC 133 specifically but I think that can be done later.

We do have a bug where we fail to turn off bracketed paste and focus reporting for commands launched from bindings,
like `bind \eg sh` for example.
But we can definitely fix that, and it's not relevant here.